base/bucket_test.go

```diff
@@ -276,3 +275,19 @@ func TestGetPoolName(t *testing.T) {
 	fakeBucketSpec.PoolName = "Liverpool"
 	assert.Equal(t, "Liverpool", fakeBucketSpec.GetPoolName())
 }
+
+func TestGetViewQueryTimeoutMs(t *testing.T) {
+	fakeBucketSpec := &BucketSpec{}
+	assert.Equal(t, uint64(0x124f8), fakeBucketSpec.GetViewQueryTimeoutMs())
+	viewQueryTimeoutSecs := uint32(0)
+	fakeBucketSpec.ViewQueryTimeoutSecs = &viewQueryTimeoutSecs
+	assert.Equal(t, uint64(0x496cebb800), fakeBucketSpec.GetViewQueryTimeoutMs())
+}
+
+func TestGetGoCBConnStringError(t *testing.T) {
+	fakeBucketSpec := &BucketSpec{Server: "sftp://localhost"}
+	str, err := fakeBucketSpec.GetGoCBConnString()
```

Comment:
The "bad scheme" error isn't originating from this function, it's actually by a library function, which might subject to change.

But overall it feels a little unessesary to have this test just to give us a single line of coverage for returning err from another function, which is otherwise well tested by `TestBucketSpec`.

Reply:
Sure; this can only add coverage of a single line; removing this;